examples/getting-started-movielens/02-ETL-with-NVTabular.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        write_hugectr_keyset=True  # only needed if using this ETL Notebook for training with HugeCTR
</pre><p>if not using HugeCTR, can we still keep this line or it should be removed? if it should be removed we can add a sentence to tell so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
added explanation